topics/materials-science/slides/introduction.html

```diff
@@ -0,0 +1,69 @@
+---
+layout: introduction_slides
+logo: "GTN"
+title: Introduction to Muon Spectroscopy
+type: introduction
+contributions:
+  authorship:
+    - anish-mudaraddi
+    - elichad
+    - patrick-austin
+    - leandro-liborio
+  editing:
+    - joelvdavies
```

Comment:
if it's for ESG, do you want to add a `funding:` key with appropriate values?

Reply:
Yes, you can add more than one :) 
Please do!

Reply:
@leandro-liborio as I understand it `funding` is a section within `contributions` alongside `authorship`, `editing`. So I don't think we associate funding directly with our individual names (at least - that wasn't how it seemed to work in the other tutorials I looked at...) and it appears in a different section of the tutorial, for example [here's the funding for a climate tutorial](https://training.galaxyproject.org/training-material/topics/climate/tutorials/ocean-data-view/tutorial.html#funding). As in that example more than one funder can be listed.

Reply:
Hi there,

I need to ask the relevant people in the lab about how to reference the  Ada Lovelace Centre and the the Physical Sciences Data Infrastructure funding.  In the meantime, Patrick have addressed all of the other issues here and we were wondering if we could merge this into the GTN.  This is because we would like to see the full thing working (the videos streaming from Galaxy YouTube, the links to the Galaxy tools working) before we present it to our muon colleagues. 

Thanks

Leandro